django/core/mail/backends/smtp.py

```diff
@@ -115,8 +115,9 @@ def _send(self, email_message):
         """A helper method that does the actual sending."""
         if not email_message.recipients():
             return False
-        from_email = sanitize_address(email_message.from_email, email_message.encoding)
-        recipients = [sanitize_address(addr, email_message.encoding)
+        encoding = email_message.encoding or settings.DEFAULT_CHARSET
+        from_email = sanitize_address(email_message.from_email, encoding)
+        recipients = [sanitize_address(addr, encoding)
                       for addr in email_message.recipients()]
```

Comment:
You can move this to the previous line (we are favoring slightly longer lines to avoid non-hanging indentation).